README.md

```diff
@@ -17,27 +19,29 @@ cd scratch-vm
 npm install
 ```
 
-## Setup
-For an extended setup example, check out the /playground directory, which includes a fully running VM instance.
-```js
-var VirtualMachine = require('scratch-vm');
-var vm = new VirtualMachine();
+## Development Server
+This requires NodeJS to be installed.
 
-// Block events
-workspace.addChangeListener(vm.blockListener);
-var flyoutWorkspace = workspace.toolbox_.flyout_.workspace_;
-flyoutWorkspace.addChangeListener(vm.flyoutBlockListener);
+For convenience, we've included a development server with the VM. This is useful because the VM can take advantage of executing in a WebWorker, which is not permitted in a local file.
 
-// Run threads
-vm.runtime.start();
+## Running The Dev Server On Linux
+Open a Terminal in the repository and run:
+```bash
+make serve
 ```
-## Development Server and Playground
-For convenience, we've included a development server with the VM. This is useful because the VM can take advantage of executing in a WebWorker, which is not permitted in a local file. To start the server, run:
 
+## Running The Dev Server On Windows
+Open a Command Prompt in the repository and run:
 ```bash
-make serve
+node_modules\.bin\webpack-dev-server --host 0.0.0.0 --content-base .\
+```
+Or
+```bash
+StartServerWindows.bat
```

Comment:
Maybe it would be better to add this to the `package.json` file:
```json
"scripts": {
  "start": "webpack-dev-server --host 0.0.0.0 --content-base ."
```
That way `npm start` would start the dev server on all platforms. Note that `npm` automatically adds `node_modules/.bin` to the PATH for the purposes of these scripts.